pages/docs/reference/whatsnew14.md

```diff
@@ -132,6 +420,7 @@ Now you can also work with Pod libraries stored locally.
 Depending on your needs, you can add dependencies between:
 * A Kotlin project and Pod libraries stored remotely in the CocoaPods repository or stored locally on your machine.
 * A Kotlin Pod (Kotlin project used as a CocoaPods dependency) and an Xcode project with one or more targets.
+* Several Kotlin Pods and an Xcode project.
```

Comment:
It seems this line is from another commit :)

Reply:
I merged changes. It's from a different branch. :)